.github/workflows/flownet.yml

```diff
@@ -56,35 +50,28 @@ jobs:
 
       - name: 📦 Install FlowNet
         run: |
-          source $VENV_PATH/bin/activate
+          pip install "matplotlib<3.2"  # As long as ERT requires it...
```

Comment:
Should we document this in the Readme as well? Or shall we assume the user will understand the error?

Reply:
I think we can simply assume the user to understand for now. Shouldn't be more than a couple of weeks before this is resolved in PyPI ERT-version (https://github.com/equinor/ert/pull/1149)